symplyphysics/laws/electricity/dissipated_heat_power_is_proportional_to_current_square.py

```diff
@@ -10,15 +13,15 @@
 ## I is current flowing through this element
 ## R is impedance of this element
 
-heat_power, current, resistance = symbols('heat_power current resistance')
-law = Eq(heat_power, current**2 * resistance)
+heat_power = joule_lenz_law.law.subs({joule_lenz_law.amount_energy: power_and_time.energy, joule_lenz_law.voltage: ohm_law.voltage })
+law = solve(heat_power, power_and_time.energy, dict=True)[0][power_and_time.energy]
```

Comment:
Write law in it's ready to use form. Derivation from other laws should follow the law. See other laws for reference.